tests/disaster-recovery/sc_arbiter/test_rgw_in_stretch.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+    polarion_id,
+    tier3,
+    red_squad,
+    rgw,
+)
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
If the test is specific to the arbiter, please add a 'stretchcluster' marker so that the test can be ignored for other deployment types

Reply:
done